deploy/start-kpng.ps1

```diff
@@ -0,0 +1,44 @@
+$ErrorActionPrefernce = "Stop"
+
+# Check for required env vars
+foreach ($var in @("CLUSTER_CIDR", "NETWORK_NAME", "NODE_IP")) {
+    if ([System.Environment]::GetEnvironmentVariable($var) -EQ $null) {
+        throw "Required envionment variable '$var' is not set!"
+    }
+}
+
+Write-Host "Importing hns.psm1"
+Import-Module C:\hpc\hns.psm1
+
+$NetworkName = $env:NETWORK_NAME
+Write-Host "Waiting for network '$NetworkName' to be available..."
+while (-Not (Get-HnsNetwork | ? Name -EQ $NetworkName)) {
+    Write-Debug "waiting for HNS network..."
+    Start-Sleep 5
+}
+Write-Host "Found HNS network '$NetworkName'"
```

Comment:
We should attempt to remove the requirement for a powershell script and move this check into kubelet. We have a check there when it is overlay, not sure if it is required for non-overlay networks.

Reply:
Can we do this in another PR?
I'll create an issue to track the work.

Reply:
agree not with in scope of this work, just wanted to bring attention to it.  Thanks!

Reply:
https://github.com/kubernetes-sigs/windows-service-proxy/issues/27 filed!